Native-Frameworks/FrameworkGridView.swift

```diff
@@ -6,12 +6,28 @@ struct FrameworkGridView: View {
     @State private var isPressed = false
     @State private var isGridviewSelected = true
     @State private var isListviewSelected = false
+    @State private var isPopupDisplayed = false
     
     var selectionView: some View {
-        GeometryReader { geo in
-            PopupSelectionView(isGridviewSelected: $isGridviewSelected, isListviewSelected: $isListviewSelected)
-                .offset(x: geo.size.width * 0.61, y: geo.size.height * 0.1)
+        HStack {
+            
+            Spacer()
+            
+            VStack {
+                PopupSelectionView(isGridviewSelected: $isGridviewSelected, isListviewSelected: $isListviewSelected)
+                
+                Spacer()
+            }
+            .padding(.top, 100)
         }
+        .frame(maxWidth: UIScreen.screenWidth, maxHeight: UIScreen.screenHeight)
```

Comment:
I don't accept this change, prefer infinity to be full screen, not you still using UIKit and we do it to remove UIKit from this project